projects/hslayers/src/common/wms/wms-get-capabilities-response.interface.ts

```diff
@@ -160,8 +160,12 @@ export interface WMSGetCapabilitiesResponse {
       Format: FormatType | FormatType[];
     };
     [_ExtendedCapabilities: string]: unknown;
-    Layer?: WmsLayer;
+    Layer?: WmsLayer; // | WmsLayer[]; ?
```

Comment:
Is this commented code for further reference at some point?

Reply:
Yes, it is mainly for future reference.
OGC OWS standard allows that (at least if I remember correctly, I can't find it now...), but I have not yet encountered any WMS service, which would implement multiple top-most layers.